test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -1432,4 +1432,41 @@ firrtl.module @Issue2315(in %x: !firrtl.vector<uint<10>, 5>, in %source: !firrtl
   // CHECK-NEXT: firrtl.mux([[EQ]], %x_4,
 }
 
+  // Check if the NLA is updated with the new lowered symbol on a field element.
+  // CHECK-LABEL: firrtl.nla @nla
+  firrtl.nla @nla [#hw.innerNameRef<@fallBackName::@test>, #hw.innerNameRef<@Aardvark::@test>, #hw.innerNameRef<@Zebra::@b>]
+  // CHECK-SAME: [#hw.innerNameRef<@fallBackName::@test>, #hw.innerNameRef<@Aardvark::@test>, #hw.innerNameRef<@Zebra::@bbundle_ready>]
```

Comment:
The naming of the new symbol is a little weird. If the symbol needs a new name, `@b_ready` or `@bundle_ready` seems better.

Reply:
Fixed, Used the normal "_" for the naming.